src/themes/light.js

```diff
@@ -2,9 +2,9 @@ export const theme = {
   colors: {
     white: 'var(--color-white)',
     black: 'var(--color-black)',
-    darkGrey: 'var(--color-gray-dark)',
-    grey: 'var(--color-gray)',
-    lightGrey: 'var(--color-gray-light)',
+    darkGrey: 'var(--color-grey-dark)',
+    grey: 'var(--color-grey)',
```

Comment:
are we calling the color gray or grey?

Reply:
Great callout! This is a typo and all these variables should be named `gray`. Can you create a separate PR to fix this? Thank you!

Reply:
I have created a separate PR